File: src/main/java/rst/wordpress/Scenario.java
package rst.wordpress;

import org.junit.After;
import org.junit.Before;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.util.concurrent.TimeUnit;

/**
 * Created by Administrator on 2016-09-23.
 */
public class Scenario {

    public WebDriver driver;
    public String baseUrl;

    @Before
    public void Setup() throws Exception {
        System.setProperty("webdriver.chrome.driver", "bin/chromedriver.exe");
        driver = new ChromeDriver();
        driver.manage().window().maximize();
        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
    }
    @After
    public void Quit(){
        driver.quit();
    }


}

File: testautomation/testautomation/Test/Scenario.cs
using System;
using System.Text;
using System.Text.RegularExpressions;
using System.Threading;
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Firefox;
using OpenQA.Selenium.Chrome;
using OpenQA.Selenium.Support.UI;

namespace PageObjectPattern
{

    public class Scenario
    {
        protected IWebDriver driver;
        protected String baseURL;
        protected StringBuilder verificationErrors;
        protected bool acceptNextAlert = false;
        protected WebDriverWait wait;
        [SetUp]
        public void Setup()
        {
            //driver = new FirefoxDriver();
            driver = new ChromeDriver();
            driver.Manage().Window.Maximize();
            driver.Manage().Timeouts().ImplicitlyWait(TimeSpan.FromSeconds(30));
            baseURL = "https://automatyzacja2016.wordpress.com/";
            verificationErrors = new StringBuilder();

            wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));

        }

        [TearDown]
        public void Teardown()
        {
            try
            {
                driver.Quit();
            }
            catch (Exception)
            {
                // Ignore errors if unable to close the browser
            }
            Assert.AreEqual("", verificationErrors.ToString());
        }

    }  
}